src/Microsoft.Azure.WebJobs.Host/Bindings/TraceWriter/TraceWriterBindingProvider.cs

```diff
@@ -10,9 +10,7 @@
 namespace Microsoft.Azure.WebJobs.Host.Bindings
 {
     /// <summary>
-    /// Binding provider handling bindings to both <see cref="TraceWriter"/> and <see cref="TextWriter"/>.
-    /// <remarks>
-    /// </remarks>
+    /// Binding provider handling bindings to <see cref="TraceWriter"/>.
```

Comment:
It still supports TextWriter though

Reply:
Oh, right. I re-added that and didn't fix the comment.